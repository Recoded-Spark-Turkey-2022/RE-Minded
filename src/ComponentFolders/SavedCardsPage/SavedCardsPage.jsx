import React from 'react';
import {Link} from "react-router-dom"
import yellowCard from './Images/YellowCard.svg';
import pinkCard from './Images/PinkCard.svg';
import blueCard from './Images/BlueCard.svg';
import leftArrow from './Images/LeftArrow.svg';
import rightArrow from './Images/RightArrow.svg';

function TicketPurchasePage() {
  return (
    <div className="flex flex-col font-poppins lg:mt-20 mt-10">
      <div className="lg:text-5xl md:text-3xl text-xl lg:ml-52 ml-10">
        <h1>YOUR SAVED CARDS</h1>
      </div>
      <div className="lg:text-xl md:text-base text-sm text-SubTexts mt-4 lg:ml-52 ml-10 lg:mr-0 mr-16">
        We only support cards as a payment method at the moment!
      </div>
      <div className="flex lg:flex-row md:flex-row flex-col self-center gap-3 lg:mt-20 mt-8 lg-ml-0 ml-10 lg:mr-0 mr-10">
        <div className="flex lg:space-y-4 self-center">
          <img
            src={leftArrow}
            alt="leftArrow"
            className="lg:rotate-0 md:rotate-0 rotate-90"
          />
        </div>
        <div>
          <img src={pinkCard} alt="pinkCard" />
        </div>
        <div>
          <img src={blueCard} alt="blueCard" />
        </div>
        <div>
          <img src={yellowCard} alt="yellowcard" />
        </div>
        <div className="flex space-y-4 self-center">
          <img
            src={rightArrow}
            alt="rightArrow"
            className="lg:rotate-0 md:rotate-0 rotate-90"
          />
        </div>
      </div>

      <div className="self-center lg:mt-12 mt-8 lg:mb-20 mb-10 pt-12 ">
        <Link to="/addcard">
          <button
            type="button"
            className="rounded-md box-border p-2 lg:pl-6 lg:pr-6 lg:text-lg md:text-base text-sm transition-all duration-250 bg-Buttons hover:bg-cyan-500 "
          >
            ADD NEW CARD +
          </button>
        </Link>
      </div>
    </div>
  );
}

export default TicketPurchasePage;
