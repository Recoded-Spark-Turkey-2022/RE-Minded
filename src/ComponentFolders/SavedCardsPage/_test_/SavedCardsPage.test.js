import React from 'react';
import renderer from 'react-test-renderer';
import SavedCardsPage from '../SavedCardsPage';

test('Saved Cards Page snapshot test', () => {
  const component = renderer.create(<SavedCardsPage />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
