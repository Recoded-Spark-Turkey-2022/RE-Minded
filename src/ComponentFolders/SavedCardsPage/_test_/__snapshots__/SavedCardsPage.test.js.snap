// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Saved Cards Page snapshot test 1`] = `
<div
  className="flex flex-col font-poppins lg:mt-20 mt-10"
>
  <div
    className="lg:text-5xl md:text-3xl text-xl lg:ml-52 ml-10"
  >
    <h1>
      YOUR SAVED CARDS
    </h1>
  </div>
  <div
    className="lg:text-xl md:text-base text-sm text-SubTexts mt-4 lg:ml-52 ml-10 lg:mr-0 mr-16"
  >
    We only support cards as a payment method at the moment!
  </div>
  <div
    className="flex lg:flex-row md:flex-row flex-col self-center gap-3 lg:mt-20 mt-8 lg-ml-0 ml-10 lg:mr-0 mr-10"
  >
    <div
      className="flex lg:space-y-4 self-center"
    >
      <img
        alt="leftArrow"
        className="lg:rotate-0 md:rotate-0 rotate-90"
        src="LeftArrow.svg"
      />
    </div>
    <div>
      <img
        alt="pinkCard"
        src="PinkCard.svg"
      />
    </div>
    <div>
      <img
        alt="blueCard"
        src="BlueCard.svg"
      />
    </div>
    <div>
      <img
        alt="yellowcard"
        src="YellowCard.svg"
      />
    </div>
    <div
      className="flex space-y-4 self-center"
    >
      <img
        alt="rightArrow"
        className="lg:rotate-0 md:rotate-0 rotate-90"
        src="RightArrow.svg"
      />
    </div>
  </div>
  <div
    className="self-center lg:mt-12 mt-8 lg:mb-20 mb-10 pt-12 "
  >
    <button
      className="rounded-md box-border p-2 lg:pl-6 lg:pr-6 lg:text-lg md:text-base text-sm transition-all duration-250 bg-Buttons hover:bg-cyan-500 "
      type="button"
    >
      ADD NEW CARD +
    </button>
  </div>
</div>
`;
